src/main/java/Model/Car.java

```diff
@@ -0,0 +1,23 @@
+package Model;
+
+public class Car {
+    private int position;
+
+    public Car(int position) {
+        this.position = position;
+    }
+
+    public boolean canMoveForward(int randomNumber) {
```

Comment:
```suggestion
    private boolean canMoveForward(int randomNumber) {
```
private으로 구현해도 되지 않을까?